static/js/show-task.js

```diff
@@ -39,20 +41,56 @@ function saveTask() {
     }});
 }
 
+function rememberState() {
+	// Finding how many number of rows are being displayed so that we can make it same after refresh
+        var dropdownElements = $('.page-list .dropdown-menu .dropdown-item');
+        var activeDropdownElement = $('.page-list .dropdown-menu .dropdown-item.active')[0];
+        for(var i=0; i < dropdownElements.length; i++)
+        {
+            if (dropdownElements[i] === activeDropdownElement) {
+                activeDropdownElementIndex = i;
+            }
+        }
```

Comment:
I think I'd rename `activeDropdownElementIndex` to `rowsPerPageDropdownIdx` or something a little more descriptive.

Also, I think this can be simplified if we use JQuery's [index()](https://api.jquery.com/index/) function:

```js
var rowsPerPageDropdownIdx = $('.page-list .dropdown-menu .dropdown-item.active').index();
```

Reply:
Thanks for the suggestion. Made the changes.